pkg/controller/service.go

```diff
@@ -102,17 +106,22 @@ func (s *Service) Run(ctx context.Context) error {
 		return fmt.Errorf("failed to setup %s controller: %w", "Deployment", err)
 	}
 
-	if s.TLSCertDir == "" {
-		logger.Info("Webhooks will be skipped. TLS certificates not present.")
```

Comment:
This was intentional by the way. See my comments elsewhere about debugging. It's pretty handy to be able to run the controller without webhooks.

Reply:
yes, this makes sense, updated to include it in.